### DISCUSSION TITLE
<title>I need to send an email immediately to anyone who could help me with - something having to do with my Android 11 - & every single web page TELLING me to turn off blocker & I never had a blocker & northing in the list has been on this cell!!! I need help very badly Pls & Ty

### DISCUSSION THREAD

**Discussion by bryanbrodyb** [community/community#78487] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I need to send an email immediately to anyone who could help me with - something having to do with my Android 11 - & every single web page TELLING me to turn off blocker & I never had a blocker & northing in the list has been on this cell!!! I need help very badly Pls & Ty

**Comment by ft10101** [comment]
try restarting your device and calm down it'll be fine in a min or so

**Comment by queenofcorgis** [comment]
Hi @bryanbrodyb , Thanks for reaching out! 

We're sorry to hear you're experiencing this issue.

GitHub is a software hosting service, we host millions of projects, so probably every phone and computer on the planet has software on it that is also hosted here.

We're not able to provide support or answers for those individual projects. GitHub Support and Community Discussions can only provide support for GitHub itself.

I'm sorry we can't help here but we'd recommend contacting a local IT specialist for assistance if you think your devices are affected by viruses or malware. They should be able to help you with these applications!
